appveyor.yml

```diff
@@ -120,11 +126,15 @@ build_script:
         -DCMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG=""
         -DCMAKE_INSTALL_PREFIX="C:/CURL"
         -DCMAKE_BUILD_TYPE=%PRJ_CFG% &&
-        cmake --build . --config %PRJ_CFG% --clean-first -- %BUILD_OPT%) else (
```

Comment:
I think you either need the `else` to make this one statement or start every `if` with a `-` to make them separate statements.

Reply:
I tried starting every if with `-`, that didn't work either. Now I'm trying nested if-elses...